app/controllers/miq_policy_controller.rb

```diff
@@ -618,7 +618,7 @@ def replace_right_cell(nodetype, replace_trees = [], presenter = nil)
       right_cell_text = if @condition.id.blank?
                           _("Adding a new %{model}") % {:model => ui_lookup(:model => 'Condition')}
                         else
-                          if @right_cell_text == @edit
+                          if @edit
```

Comment:
This is not the same as the original code.  The original was also setting `@right_cell_text`, but this is no longer being set here.